MANUAL.txt

```diff
@@ -3141,6 +3141,20 @@ equivalent of the Markdown in the `abstract` field:
     <p>This is the abstract.</p>
     <p>It consists of two paragraphs.</p>
 
+When writing DOCX or ODT, first-level metadata with a String value will be
+included in the resulting document as Document Properties or Custom
+Properties accordingly. The name of the metadata will be the same
+as the Document/Custom Property, except in the following cases:
+
+`subtitle`
```

Comment:
"subtitle" <-> "subject" is no longer an exception, this association has been removed.